contrib/parseq-lambda-names/pom.xml

```diff
@@ -0,0 +1,159 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.linkedin.parseq</groupId>
+    <artifactId>parseq-lambda-names</artifactId>
+    <version>2.6.8</version>
+
+    <dependencies>
+        <dependency>
+        <groupId>com.linkedin.parseq</groupId>
+            <artifactId>parseq</artifactId>
+            <version>2.6.8</version>
+            <!-- TODO: this needs to be improved -->
+            <scope>system</scope>
+            <systemPath>${basedir}/../../dist/parseq-${version}.jar</systemPath>
```

Comment:
Thats because TaskDescriptor is defined in parseq main project. ASMBasedTaskDescriptor implements that.